editor/app/ems/[incident_number]/page.tsx

```diff
@@ -134,6 +134,9 @@ export default function EMSDetailsPage({
       crash_pks: unmatchedTimeInterval[0] ? allCrashPks : relatedCrashPks,
     },
     typename: "people_list_view",
+    options: {
+      keepPreviousData: false,
```

Comment:
the associated people list was not getting new data after you set a multiple matches crash status to unmatched by review Q/A. i expected the results to update to be 0 but you had to refresh the page to see that reflected, even if i refetched it in the `onSaveCallback`. updating this setting did the trick, but is this okay?